examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -410,7 +420,96 @@ def act(self, obs, env):
         if self._beg_state_lock:
             self.current_state = FetchState.BEG_RESET
 
-        if self.current_state == FetchState.SEARCH:
+        if self.current_state == FetchState.WAIT:
+            self.current_state = FetchState.FOLLOW
+            self._init_policy_input()
+
+        if self.current_state == FetchState.FOLLOW:
+            # This is the following state, in which the robot tries to follow the human
+            # There are two cases here. When the human is far away from the robot, we use
+            # normal point nav, and when the human is close enough, we switch to the
+            # social nav. This allows us to effectively find and follow the human
+            if self.should_start_skill:
+                human_robot_geo_dis = self._habitat_env._sim.geodesic_distance(
+                    robot_trans, human_trans
```

Comment:
Nit: `trans` is misleading as it could mean either translation or transform. I would use `pos` instead.

Reply:
done, thank you!